packages/e2e/features/ui/components/authenticator/sign-in-with-phone.feature

```diff
@@ -10,6 +10,24 @@ Feature: Sign In with Phone Number
   Background:
     Given I'm running the example "ui/components/authenticator/sign-in-with-phone"
 
+  @angular @react @vue
+  Scenario: Reset Password with valid phone with country code
+    When I click the "Forgot your password?" button
+    When I select my country code with status "CONFIRMED"
+    And I type my "phone number" with status "CONFIRMED"
+    And I intercept '{ "headers": { "X-Amz-Target": "AWSCognitoIdentityProviderService.ForgotPassword" } }' with fixture "reset-password"
+    And I click the "Send code" button
+    And I verify the body has "+19995554444" included
```

Comment:
I check the username twice in the body to verify to make sure that it's being sent correctly over and not missing the country_code.